internal/service/credentials/resource_credential_issuance_rule_test.go

```diff
@@ -136,22 +201,6 @@ func TestAccCredentialIssuanceRule_Full(t *testing.T) {
 		CheckDestroy:             testAccCheckCredentiaIssuanceRuleDestroy,
 		ErrorCheck:               acctest.ErrorCheck(t),
 		Steps: []resource.TestStep{
-			// full
```

Comment:
I think I actually prefer this setup of tests - it means there is a consistent variable of what a "fullStep" output looks like vs a "minimalStep".  If they get broken out into separate tests, it increases the work to maintain when we add or remove new attributes

Reply:
Will revert.